components/slider/index.css

```diff
@@ -516,22 +515,6 @@ governing permissions and limitations under the License.
   }
 }
 
-
-.spectrum-Slider {
-  &.is-disabled {
-    cursor: default;
-
-    .spectrum-Slider-handle {
-      cursor: default;
-      pointer-events: none;
-    }
-
-    .spectrum-Slider-tickLabel {
-      color: var(--highcontrast-slider-label-text-color-disabled, var(--mod-slider-label-text-color-disabled, var(--spectrum-slider-label-text-color-disabled)));
-    }
-  }
-}
```

Comment:
Relocated this to an existing `spectrum-Slider.is-disabled` selector on new line 628